### DISCUSSION TITLE
Display position at bottom regardless of compass orientation?

### DISCUSSION THREAD

**Discussion by tommycrock** [osmandapp/OsmAnd#15795] [Q&A]
My memory is that centring on your position with the compass on fixed/north centred the screen on you regardless of the display position setting - I'm currently finding it places the map at the bottom despite being oriented north. Is that as designed? It makes sense with movement/compass orientation to be able to set it bottom.
Thanks

**Comment by vshcherb** [comment]
Yes, it's a separate setting now - and we display position at bottom regardless of compass orientation.
It makes sense to make it easier configurable without weird settings name and other UX issues

**Comment by jfburke3** [comment]
@vshcherb To be sure I understand:  if you  have the map orientation set so that North is fixed toward the top of the screen, and you are navigating to a location south of your current position, your location marker will always be near the bottom of the screen?

**Reply by tommycrock** [reply]
It depends on your selection in: 
settings, [this profile], general settings, "display position" 
which can be central or bottom.
Whether you are navigating or not wouldn't affect it but you could produce this odd effect. I don't understand the change of behaviour, although I appreciate that the name of the setting was awkward before.

**Reply by dmpr0** [reply]
Yes

**Comment by tommycrock** [comment]
Thanks for changing it again! (There is now an 'auto' option)
